tests/unit/coordinator/test_list.py

```diff
@@ -239,7 +245,7 @@ def test_api_list_deduplication(backup_manifest: BackupManifest, tmp_path: Path)
                         name="1",
                         start=datetime.datetime(2020, 1, 2, 3, 4, 5, 678, tzinfo=datetime.timezone.utc),
                         end=datetime.datetime(2020, 1, 2, 5, 6, 7, 891, tzinfo=datetime.timezone.utc),
-                        plugin=Plugin("clickhouse"),
```

Comment:
is this change actually necessary, given the fact that we've moved things around and no longer require list to parse the plugin manifest?